.github/workflows/vitest-example-test-suite.yaml

```diff
@@ -0,0 +1,43 @@
+name: vitest-example-test-suite
```

Comment:
I can move this to the circleci file if that's preferable, but I'm not super familiar with the setup there and may need some guidance